src/PatientChecking/PatientChecking/Controllers/AppointmentController.cs

```diff
@@ -14,10 +15,12 @@ namespace PatientChecking.Controllers
     public class AppointmentController : BaseController
     {
         private readonly IAppointmentService _appointmentService;
+        private readonly INotyfService _notyf;
 
-        public AppointmentController(IAppointmentService appointmentService)
+        public AppointmentController(IAppointmentService appointmentService, INotyfService notyf)
```

Comment:
Should rename `notyf` to `notyfService`

Reply:
I have removed this since I use ViewMessageHelper instead of INotyService